mailet/icalendar/src/main/resources/ical4j.properties

```diff
@@ -0,0 +1,5 @@
+ical4j.unfolding.relaxed=true
+ical4j.parsing.relaxed=true
+ical4j.validation.relaxed=true
+ical4j.compatibility.outlook=true
+ical4j.compatibility.notes=true
```

Comment:
I don't. That will be hard to be sure that this file is well deployed, while with code we are sure...